server/src/test/java/io/crate/analyze/EvaluatingNormalizerTest.java

```diff
@@ -162,4 +164,21 @@ public void testEvaluationClusterGranularity() {
         Symbol query = visitor.normalize(op_or, coordinatorTxnCtx);
         assertThat(query).isExactlyInstanceOf(Function.class);
     }
+
+    @Test
+    public void test_normalize_function_with_alias_symbol_as_argument() {
+        EvaluatingNormalizer visitor = new EvaluatingNormalizer(nodeCtx, RowGranularity.CLUSTER, null, null);
+
+        // (col AS alias) = 1
+        Function func = new Function(
+            EqOperator.SIGNATURE,
+            List.of(
+                new AliasSymbol("alias", createReference("col", DataTypes.INTEGER)),
+                Literal.of(1)
+            ),
+            DataTypes.BOOLEAN
+        );
+        Function normalized = (Function) visitor.normalize(func, coordinatorTxnCtx);
+        assertThat(normalized.toString()).isEqualTo("(col = 1)");
+    }
```

Comment:
I think we can add a comment that it supersedes tests removed in https://github.com/crate/crate/commit/7f763496099d2daa51fa1734da2fada32bad265f to reflect the fact that un-aliasing is done in normalizer since https://github.com/crate/crate/pull/12474/commits/b2a5b01c6deefe93cdbf0c0005e29422e7838887